src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -504,7 +517,9 @@ export class InlineChatWidget {
 		this.updateInfo('');
 
 		this._elements.accessibleViewer.classList.toggle('hidden', true);
-
+		this._codeBlockModelCollection.clear();
```

Comment:
IMO clearing of the code block collection should happen inside `updateChatMessage`